apps/web/src/pages/Profile/ExperienceAndSkillsPage/components/AddExperienceDialog.tsx

```diff
@@ -0,0 +1,257 @@
+    },
+  ];
+
+  const Footer = React.useMemo(
+    () => (
+      <Dialog.Close>
+        <IconButton color="purple" icon={ArrowLeftIcon}>
+          {intl.formatMessage({
+            defaultMessage: "Cancel and go back",
+            id: "tiF/jI",
+            description: "Close dialog button",
+          })}
+        </IconButton>
+      </Dialog.Close>
+    ),
+    [intl],
+  );
+
+  return (
+    <Dialog.Root defaultOpen={defaultOpen}>
+      <Dialog.Trigger>
+        <IconButton color="blue" icon={PlusIcon}>
+          {intl.formatMessage({
+            defaultMessage: "Add a new experience",
+            id: "bOAF9o",
+            description:
+              "Button to open modal to add a new experience to the profile",
+          })}
+        </IconButton>
+      </Dialog.Trigger>
+      <Dialog.Content>
+        <Dialog.Header
+          color="black"
+          subtitle={intl.formatMessage({
+            defaultMessage:
+              "Get started by selecting the type of experience you would like to add.",
+            id: "h5MtHc",
+            description: "Subtitle for the Add Experience dialog",
+          })}
+        >
+          {intl.formatMessage({
+            defaultMessage: "Add a new experience to your resume",
+            id: "yI5DFC",
+            description: "Heading for the Add Experience dialog",
+          })}
+        </Dialog.Header>
+        <div data-h2-margin="base(x1, 0)">
+          {intl.formatMessage({
+            defaultMessage:
+              "Before you start describing your experience, first we would like to know what category it falls into. There are five options to choose from, you can add as many experiences to each category as you want.",
+            id: "MhgtB6",
+            description: "Instructions for the Add Experience dialog",
+          })}
+        </div>
+        <div
+          data-h2-margin="base(0, x0.5)"
+          data-h2-flex-grid="base(stretch, x2, x1)"
+        >
+          {experienceSections.map((section) => (
+            <div
+              key={section.title}
+              data-h2-margin-bottom="base(x2)"
+              data-h2-flex-item="base(1of1) p-tablet(1of2)"
+            >
+              <div
+                data-h2-display="base(flex)"
+                data-h2-flex-direction="base(column)"
+                data-h2-justify-content="base(space-between)"
+                data-h2-height="base(100%)"
+              >
+                <div>
+                  <div data-h2-margin="base(x0.5, 0)">
+                    <strong>
+                      <span
+                        data-h2-display="base(inline-block)"
+                        data-h2-height="base(x1)"
+                        data-h2-width="base(x1)"
+                        data-h2-vertical-align="base(middle)"
+                        data-h2-margin="base(0, x0.5, 0, 0)"
+                      >
+                        <section.icon />
```

Comment:
You can update this to `PascalCase` by deconstructing it in the map function.

```tsx
{experienceSections.map(({icon: Icon, ...section}) => ( ... )}
```

Reply:
:+1: [alias section icon](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5701/commits/28ffa06729d044708c42d05f5dff8e77b25a85dc)